src/NNlib.jl

```diff
@@ -5,6 +5,8 @@ using Requires
 export σ, sigmoid, relu, leakyrelu, elu, swish, selu, softplus, softsign, logσ, logsigmoid,
   softmax, logsoftmax, maxpool, meanpool
 
+nnpack_available=(is_linux() || is_apple()) && isfile("libnnpack.so")
```

Comment:
Just inline this variable if it's only used once. Also, this should look in the `deps` directory